images/docker-selenium/README.md

```diff
@@ -25,4 +25,73 @@ docker pull cgr.dev/chainguard/docker-selenium:latest
 ```
 <!--getting:end-->
 
-<!--body:start--><!--body:end-->
+<!--body:start-->
+
+## Usage
+
+Run the container with the following command:
+
+```bash
+docker run --shm-size="2g" -p 4444:4444 -p 7900:7900 cgr.dev/chainguard/docker-selenium:latest
+```
+
+Verify Selenium is running:
+
+```bash
+curl -sL http://localhost:4444/status` should return a `200` status code, and the response body should contain `{"value":{"ready":true}}` with 1 `availability: UP` node.
+```
+
+Access Selenium:
+
+```bash
+http://localhost:4444
+```
+
+Access noVNC:
+
+```bash
+http://localhost:7900/vnc.html`:
+```
+
+* Login with the password `secret`.
+
+Any WebDriver tests should point to Selenium on on port 4444.
+
+The following is an example ChromeDriver test using Selenium to retrieve Chainguard's homepage:
+
+```python
+from selenium import webdriver
+from time import sleep
+
+print("Testing ChromeDriver...")
+
+options = webdriver.ChromeOptions()
+
+driver = webdriver.Remote(
+  command_executor='http://localhost:4444/wd/hub',
+  options=options
+)
+
+'''Maximize window'''
+driver.maximize_window()
+sleep(5)
+
+'''Retrieve Chainguards homepage'''
+driver.get("https://www.chainguard.dev/")
+sleep(5)
+
+'''Exit'''
+driver.close()
+driver.quit()
+print("ChromeDriver test successful!")
+```
+
+Please note that Chromium is ran in a headless state with the sandbox and GPU access disabled with the flags:
```

Comment:
Can we look at expanding this section a little with the reasoning? i.e the reason we need to run in sandbox and with GPU disabled, being a container.. etc

Reply:
Definitely!